docs/releasehistory.rst

```diff
@@ -86,8 +101,8 @@ New features
 - Add ``restrain_atoms`` to restrain molecule conformation through an harmonic restrain
   (`#255 <https://github.com/choderalab/openmmtools/issues/255>`_)
 
-Bugfixes
---------
+Bug fixes
```

Comment:
Let's also add that this contains bugfix https://github.com/choderalab/openmmtools/pull/311
Apologies for not adding that earlier in the PR!

Reply:
I do that on this bullet [here](https://github.com/choderalab/openmmtools/pull/313/files#diff-7c41e54961aec78536c96ae2dfe11981R9)